### PR TITLE
MBM-68 feature | add energized iron ingot crafting recipe

### PR DESCRIPTION
**Pull request by solidwaterslayer**

### CODE REVIEW

**Comment by MattKirchh** [comment]
Not only are there supposed to be 2 crafting recipes for the energized iron ingot, but the 1 recipe you have implemented here is wrong. Please carefully look at the description of the Jira item.

File: src/main/resources/data/minebound/recipes/energized_iron_ingot.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "ingredients": [
+    {
+      "item": "minebound:energized_iron_block"
+    }
+  ],
+  "result": {
+    "item": "minebound:energized_iron_ingot",
+    "count": 9
+  },
+  "type": "minecraft:crafting_shapeless"
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/recipes/energized_iron_ingot.json:11]: Should put the `type` field at the top for consistency.

File: src/main/resources/data/minebound/recipes/energized_iron_ingot_from_alloy_furnace.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "cookingtime": 200,
+  "experience": 0.7,
+  "ingredient": {
+    "item": "minebound:energized_iron_ingot"
+  },
+  "result": "minebound:energized_iron_block",
+  "type": "minebound:alloy_furnace"
+}
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/recipes/energized_iron_ingot_from_alloy_furnace.json:9]: This is not a recipe we defined. This is:

```
{
  "type": "minebound:alloy_furnace",
  "ingredients": [
    {
      "item": "minecraft:redstone"
    },
    {
      "item": "minecraft:iron_ingot"
    }
  ],
  "output": {
    "item": "minebound:energized_iron_ingot"
  },
  "experience": 1.0,
  "cookingtime": 200
}
```

File: src/main/resources/data/minebound/recipes/energized_iron_ingot_from_alloy_furnace.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "type": "minebound:alloy_furnace",
+  "ingredients": [
+    {
+      "item": "minebound:iron_ingot"
+    },
+    {
+      "item": "minebound:redstone"
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/recipes/energized_iron_ingot_from_alloy_furnace.json:8]: Should be `minecraft:`, not `minebound:`.

**Review by YonatanWN — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]

**Review by MattKirchh — APPROVED** [review]

**Pull request merged**